src/usr/local/www/interfaces.php

```diff
@@ -2800,6 +2812,13 @@ function build_port_list() {
 	$pconfig['provider']
 ))->setHelp('This field can usually be left empty.');
 
+$section->addInput(new Form_Input(
+	'pppoe_host-uniq',
+	'Host-Uniq',
+	'text',
+	$pconfig['pppoe_host-uniq']
+))->setHelp('This field can usually be left empty.');
```

Comment:
Needs a better description of what the field is and why someone might need to set it.